react/legacy/index.js

```diff
@@ -0,0 +1,141 @@
+import React, { Fragment, useEffect, useLayoutEffect, useState } from 'react'
```

Comment:
This file was just moved from `react` to `react/legacy`, please refrain from looking too deeply into it for this PR